web/src/styles/globals.css

```diff
@@ -73,6 +73,11 @@
     --sidebar-accent-foreground: 240 5.9% 10%;
     --sidebar-border: 220 13% 91%;
     --sidebar-ring: 217.2 91.2% 59.8%;
+
+    --chart-1: 176 60.8% 19%;
+    --chart-2: 217 91.2% 59.8%;
+    --chart-3: 43 96% 40%;
+    --chart-4: 0 72.2% 50.6%;
```

Comment:
Those map to
- dark-green
- dark-blue
- dark-yellow
- dark-red
If there is a good way to reuse the values instead, feel free to recommend it. Same thing below for the dark mode.